pkg/agent/billing.go

```diff
@@ -250,7 +252,7 @@ func (s *billingMetricsState) drainAppendToBatch(conf *BillingConfig, batch *bil
 			EndpointID:     key.endpointID,
 			StartTime:      s.pushWindowStart,
 			StopTime:       now,
-			Value:          int(math.Round(history.total.activeTime.Seconds())),
+			Value:          math.Round(history.total.activeTime.Seconds()),
```

Comment:
```suggestion
			Value:          history.total.activeTime.Seconds(),
```

We can drop the `math.Round` now that we don't need to convert to an integer :)